src/server/req_runjob.c

```diff
@@ -1310,6 +1334,16 @@ post_sendmom(struct work_task *pwt)
 				        "send of job to %s failed error = %d",
 				        jobp->ji_qs.ji_destin, pbs_errno);
 			log_event(PBSEVENT_JOB, PBS_EVENTCLASS_JOB, LOG_INFO, jobp->ji_qs.ji_jobid, log_buffer);
+			snprintf(log_buffer, sizeof(hook_buf),
+					"Not Running: PBS Error: %s", pbse_to_txt(PBSE_MOMREJECT));
```

Comment:
They are same. I have attached log in the jira for how it looks like in both the scenarios.